movement/io/load_poses.py

```diff
@@ -0,0 +1,58 @@
+import logging
+from pathlib import Path
+from typing import Optional, Union
+
+import pandas as pd
+
+from movement.io.validators import DeeplabcutPosesFile
+from movement.log_config import configure_logging
+
+# initialize logger
+configure_logging()
```

Comment:
If another package calls `movement` and has it's own logging, does having this call mean there ends up being two log files? Need to make sure this doesn't interfere with the users' own logging.

Reply:
Now I changed several things about logging.

1. I moved the call to `configure_logging()` in `__init__.py`, so that it runs upon import. It doesn't make sense to configure it for every script/module separately, and there is no other single entry point to use.
2. I want to make sure that a logger with the name 'movement' is configured only once, so I added the following safeguard:
```python
# If a logger with the given name is already configured
    if logger_name in logging.root.manager.loggerDict:
        # skip configuration
        return
```

To my understanding, this should not interfere with other logs by the user, but would be grateful if you could take a look at the updated `configure_logging()` function.